target/linux/ar71xx/files/arch/mips/ath79/mach-anonabox-pro.c

```diff
@@ -0,0 +1,111 @@
+/*
+ * Atheros ANONABOX_PRO reference board support
```

Comment:
Anonabox Pro doesn't seem to be an Atheros design/product. And it also doesn't seem to a any kind of reference design. At least this https://www.anonabox.com/buy-anonabox-pro.html advertises it as normal product.